src/binding.cc

```diff
@@ -1503,6 +1503,14 @@ const v8::Script* v8__Script__Compile(const v8::Context& context,
                           const_cast<v8::ScriptOrigin*>(&origin)));
 }
 
+const v8::UnboundScript* v8__Script__GetUnboundScript(const v8::Script& script) {
+  return local_to_ptr(ptr_to_local(&script)->GetUnboundScript());
+}
+
+const v8::Script* v8__UnboundScript__BindToCurrentContext(const v8::UnboundScript& unbound_script) {
```

Comment:
Can you wrap lines at 80 columns? Thanks.

Reply:
Done!